scripts/verify-defaults.sh

```
@@ -0,0 +1,139 @@
+    exit 1
+fi
+
+echo "Comparing settings defaults..."
+echo "  Forklift:    $FORKLIFT_DEFAULTS"
+echo "  kubectl-mtv: $KUBECTL_MTV_TYPES"
+echo ""
+
+# Temporary files for comparison
+FORKLIFT_SETTINGS=$(mktemp)
+KUBECTL_MTV_SETTINGS=$(mktemp)
+trap "rm -f $FORKLIFT_SETTINGS $KUBECTL_MTV_SETTINGS" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)